tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -356,14 +362,46 @@ def switch_to_window(self, wptrunner_id):
         """Switch to a window given a wptrunner window id
 
         :param str wptrunner_id: window id"""
-        pass
-
-    def switch_to_frame(self, index):
+        if wptrunner_id is None:
+            return
+
+        stack = [str(item) for item in self.parent.base.window_handles()]
```

Comment:
Is it possible for `window_handles()` to not return strings (and if not, what are they)?

Reply:
I think I got integer window handles back from Firefox. This is a bug per spec.

Reply:
Ack; could you file a bug against the Firefox case?

Reply:
Now I look firefox is returning integers as strings, so it's possible I was just confused.